national-institute-of-technology-karnataka.csl

```diff
@@ -1,38 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="en-GB">
   <info>
-    <title>NITK Thesis</title>
+    <title>National Institute of Technology Karnataka</title>
     <title-short>NITK</title-short>
     <id>http://www.zotero.org/styles/nitk-thesis</id>
```

Comment:
@rmzelle . Thanks for patient and sincere review. Updated the latest correction. Please let me know if I need to change something.